README.md

```diff
@@ -102,6 +117,14 @@ attempt to create it.
 It is the Administrator's responsibility to ensure there is sufficient
 space for the global log.
 
+### Enabling debug for various components
+
+The runtime, the shim (`cc-shim`), and the hypervisor all have separate debug
+options in the configuration file (see the [configuration](#Configuration) file).
```

Comment:
This reads as a little bit repetitive. Can we instead have the first "configuration" file as the link as it is on line 89?

Reply:
Good catch - fixed.